lib/github/client.js

```diff
@@ -32,87 +32,108 @@
 
 'use strict';
 
-const util = require('util');
-
 const Gofer = require('gofer');
 
+const pkg = require('../../package');
+
 const parseRepository = require('./parse-repository');
 
-function Github(config) {
-  Gofer.call(this, config, 'github');
-}
+class Github extends Gofer {
+  constructor(config) {
+    super(config, 'github', pkg.version, pkg.name);
+  }
 
-util.inherits(Github, Gofer);
-Github.prototype.registerEndpoints({
-  pull: function pull(request) {
+  get pull() {
     return {
-      commits: function commits(pullId) {
-        return request('/repos/{owner}/{repo}/pulls/{pullId}/commits', {
-          pathParams: {
-            pullId,
-          },
-        }).json();
-      },
-      get: function get(pullId) {
-        return request('/repos/{owner}/{repo}/pulls/{pullId}', {
-          pathParams: {
-            pullId,
-          },
-        }).json();
-      },
+      /**
+       * @param {string} pullId
+       * @return {Promise<any>}
+       */
+      get: pullId =>
+        this.get('/repos/{owner}/{repo}/pulls/{pullId}', {
+          pathParams: { pullId },
```

Comment:
```suggestion
          pathParams: { pullId },
          endpointName: 'pull.get',
```

ideally, stick `endpointName` on all of these (one downside of the class syntax)